ibtk/include/ibtk/FEDataManager.h

```diff
@@ -324,7 +319,12 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
     static void freeAllManagers();
 
     /*!
-     * \brief Register a load balancer for non-uniform load balancing.
+     * \brief Register a load balancer for non-uniform load balancing. This
+     * function must be called to set up the workload index before
+     * updateWorkloadEstimates is called.
+     *
+     * @deprecated Since the correct workload index is passed in via
+     * addWorkloadEstimate this function is no longer necessary.
```

Comment:
We never used the load balancers anywhere (and they should only be used by the parent hierarchy integrator) so all of these are deprecated. In principle, a non-hierarchy doesn't need to even store a pointer to any samrai object: it just needs to know how to write its workload estimate into the given hierarchy and variable index.

Reply:
You know, I really think it would be OK to just drop this interface --- I would be shocked if anyone was using it.

Reply:
I meant to ask you about that but I forgot: I cannot forsee any reason that any of these objects would need the load balancer. It sounds like you are not aware of any either.